docs/source/tutorial_basic.md

```diff
@@ -171,15 +171,13 @@ Another simple world we include is MultiAgentDialogWorld, which is
 similar but generalizes this to cycle between any number of agents in a
 round robin fashion.
 
-### Advanced Worlds
-
-We also include a few more advanced "container" worlds: in particular,
-we include both a BatchWorld and a HogwildWorld. These worlds are
-automatically used when either the `numthreads` parameter or the
-`batchsize` parameter are set to greater than one. Some extra
-functionality is needed to get these to work on the side of both the
-teacher and the learner, but we'll cover that in a different tutorial
-(see: tutorial\_worlds).
+:::{note} Advanced Worlds
+We also include a few more advanced "container" worlds: in particular, we
+include both a BatchWorld and a DyamicBatchWorld. These worlds may be used when
+certain options are sent. See the [tutorial\_worlds](Worlds) tutorial to
```

Comment:
I think this was supposed to be a link. The link here is not active in my doc.

Reply:
same, link doesnt wort

Reply:
Thanks. We have an automatic link checker in the CI tests. Will fix.